addons/im_livechat/static/src/models/public_livechat_global.js

```diff
@@ -1,8 +1,44 @@
 /** @odoo-module **/
 
 import { registerModel } from '@mail/model/model_core';
+import { attr } from '@mail/model/model_field';
+
+import utils from 'web.utils';
 
 registerModel({
     name: 'PublicLivechatGlobal',
     identifyingFields: ['messaging'],
+    lifecycleHooks: {
+        _created() {
+            // History tracking
+            const page = window.location.href.replace(/^.*\/\/[^/]+/, '');
```

Comment:
➡️ `window.location.pathname`?

Reply:
What do you mean by this change?
This LOC is just a move, so I'm not entirely sure it's equivalent to `window.location.pathname`.

The "?" at end of your comment gives me the impression you are also unsure.

Keep in mind lots of LOCs in public livechat I still don't get them.
In the meantime, I do structural changes (i.e. basically moves and removal of dead/useless code), so if we are unsure, better keep the exact same LOCs.